packages/wp-story-editor/src/index.js

```diff
@@ -59,6 +60,7 @@ import { GlobalStyle } from './theme.js';
 
 window.webStories = window.webStories || {};
 window.webStories.domReady = domReady;
+window.webStories.data = data;
```

Comment:
This may allow third-party plugin developers to use data like

```js
webStories.data.select( 'sliceName' );
webStories.data.dispatch('sliceName', 'action', value);
webStories.data.subscribe();
```